app/ui-admin/client/admin.js

```diff
@@ -421,17 +421,21 @@ Template.admin.events({
 			});
 		});
 	},
-	'click .rc-header__section-button .save'() {
+	'click .rc-header__section-button .save'(e) {
+		e.target.disabled = true;
+		e.target.innerText = TAPi18n.__('Saving');
 		const group = FlowRouter.getParam('group');
 		const query = { group, changed: true };
 		const rcSettings = TempSettings.find(query, { fields: { _id: 1, value: 1, editor: 1 } }).fetch() || [];
 		if (rcSettings.length === 0) {
+			e.target.innerText = TAPi18n.__('Save_changes');
 			return;
 		}
 
 		const failedSettings = [];
 
 		settings.batchSet(rcSettings, (err) => {
+			e.target.innerText = TAPi18n.__('Save_changes');
```

Comment:
and here

Reply:
In the old version, when the settings is saved the button is change to disabled until you change some input.

If i add `disabled false` the button is forced to change, and "stop to saving"  until you change some input